docker-compose.yml

```diff
@@ -20,7 +20,10 @@ services:
       context: ./
     restart: unless-stopped
     volumes:
-      - ./:/kb
+      - ./knowledge-base:/knowledge-base
+      - ./repo-path:/knowledge-base
+      - ./ostis-example-app.ini:/knowledge-base
+      - ./problem-solver:/knowledge-base
```

Comment:
it still loads entire problem solver just to build 2 files from it
and in the end it fails to build
![image](https://github.com/user-attachments/assets/3148ebba-4202-462f-a712-32a26c92f1c8)

Reply:
I left TODO in docker-compose.yml and issue https://github.com/ostis-apps/ostis-example-app/issues/54